src/app/view/perspectives/post-it/add-document/add-document.component.ts

```diff
@@ -52,10 +52,20 @@ export class PostItAddDocumentComponent implements OnInit, OnDestroy {
     this.createPostIt.emit({
       collection: this.selectedCollection,
       collectionCode: this.selectedCollection.code,
-      data: {}
+      data: this.dataWithAttributeNames()
     });
   }
 
+  private dataWithAttributeNames(): { [attributeName: string]: string } {
+    let result = {};
+
+    this.selectedCollection.attributes.forEach(attribute => {
+      result[attribute.name] = '';
```

Comment:
We must store all attributes under `fullName` a.k.a. `id` key otherwise there might be some conflicts.